lib/wallaroo/core/sink/connector_sink/connector_sink_notify.pony

```diff
@@ -23,8 +23,12 @@ use "wallaroo/core/common"
 use "wallaroo/core/network"
 use "wallaroo_labs/bytes"
 use cp = "wallaroo_labs/connector_protocol"
+use "wallaroo_labs/logging"
 use "wallaroo_labs/mort"
 
+// use @ll[I32](sev_cat: U16, fmt: Pointer[U8] tag, ...)
```

Comment:
Delete this?

Reply:
Yes.  It's a leftover from a compiler feature/bug-perhaps that I don't understand fully.  The C FFI type declaration is required IIRC in connector_sink.pony but triggers a complaint if present in connector_sink_notify.pony.